src/request/components/RequestDetailPanelDataOperationCommand.tsx

```diff
@@ -0,0 +1,17 @@
+import React = require('react');
+import Highlight = require('react-highlight');
+
+export interface IRequestDetailPanelDataOperationCommandProps {
+    command: string,
+    language: string
+}
+
+export class RequestDetailPanelDataOperationCommand extends React.Component<IRequestDetailPanelDataOperationCommandProps, {}> {
+    public render() {
+        return (
+            <div className='tab-data-operation-command'>
+                <Highlight language={this.props.language}>{this.props.command}</Highlight>
+            </div>
+        );
+    }
+}
```

Comment:
The "parent" component is a "dumb" component (in the `redux` sense); it knows where to place the component (in the left master/detail/split panel) but doesn't know how to pass it the correct set of properties (because those are determined by application state and set by the table component).